src/browser/core_state.ts

```diff
@@ -125,7 +125,7 @@ export function getEntityInfo(identity: Shapes.Identity) {
 }
 
 export function getExternalOrOfWindowIdentity(identity: Shapes.Identity): Shapes.ProviderIdentity|undefined {
-    const { uuid, name } = identity;
```

Comment:
maybe rename to get identity & entity type?  Name no longer really applicable.

Reply:
Good idea. Renamed to getEntityIdentity as it still returns identity but not only for windows.